config/zope-product/tox.ini.j2

```diff
@@ -38,38 +38,7 @@ commands =
 %(line)s
 {% endfor %}
 
-{% include 'tox-release-check.j2' %}
```

Comment:
This line has to stay, right?

Reply:
No, the next line includes `tox-lint.j2` which already includes this file at the top, see config/default/tox-lint.j2

So either we have to remove it here or in `tox-lint.j2` but then add it in the places where `tox-lint.j2` is included.